packages/pebble-shared/svgs/person-outline.svg

```diff
@@ -0,0 +1,5 @@
+<svg width="1000" height="1000" viewBox="0 0 1000 1000" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path d="M500 420.833C548.325 420.833 587.5 381.658 587.5 333.333C587.5 285.008 548.325 245.833 500 245.833C451.675 245.833 412.5 285.008 412.5 333.333C412.5 381.658 451.675 420.833 500 420.833Z" fill="#000"/>
+<path d="M500 620.833C376.25 620.833 245.833 681.666 245.833 708.333V754.166H754.166V708.333C754.166 681.666 623.75 620.833 500 620.833V620.833Z" fill="#000"/>
```

Comment:
something wrong with this icon. there is slight gap between svg paths

<img width="295" alt="Screenshot 2023-03-06 at 11 56 55 AM" src="https://user-images.githubusercontent.com/8132723/223034865-0a029462-16a2-4a4a-bc53-8e078186e7c2.png">